fs/sync/sync.go

```diff
@@ -522,13 +522,16 @@ func copyEmptyDirectories(f fs.Fs, entries map[string]fs.DirEntry) error {
 	return nil
 }
 
-func parentDirCheck(entries map[string]fs.DirEntry, entry fs.DirEntry) {
```

Comment:
The should probably be called `srcParentDirCheck` now it only acts on the source directory.